diff --git a/Common/sources/constants.js b/Common/sources/constants.js
index 0663ead..490582a 100644
--- a/Common/sources/constants.js
+++ b/Common/sources/constants.js
@@ -75,7 +75,7 @@ exports.LICENSE_RESULT = {
   ExpiredLimited: 11
 };
 
-exports.LICENSE_CONNECTIONS = 20;
+exports.LICENSE_CONNECTIONS = 9999;
 exports.LICENSE_EXPIRE_USERS_ONE_DAY = 24 * 60 * 60; // day in seconds
 
 exports.AVS_OFFICESTUDIO_FILE_UNKNOWN =  0x0000;
diff --git a/Common/sources/license.js b/Common/sources/license.js
index c273afe..8786f62 100644
--- a/Common/sources/license.js
+++ b/Common/sources/license.js
@@ -45,20 +45,20 @@ exports.readLicense = function*() {
 		count: 1,
 		type: c_LR.Success,
 		light: false,
-		packageType: constants.PACKAGE_TYPE_OS,
+		packageType: constants.PACKAGE_TYPE_I,
 		mode: constants.LICENSE_MODE.None,
-		branding: false,
+		branding: true,
 		connections: constants.LICENSE_CONNECTIONS,
-		customization: false,
-		usersCount: 0,
+		customization: true,
+		usersCount: constants.LICENSE_CONNECTIONS,
 		usersExpire: constants.LICENSE_EXPIRE_USERS_ONE_DAY,
-		hasLicense: false,
-		plugins: false,
+		hasLicense: true,
+		plugins: true,
 		buildDate: oBuildDate,
 		startDate: startDate,
-		endDate: null,
-		customerId: ""
+		customerId: "",
+		endDate: new Date("2099-01-01T23:59:59.000Z")
 	}, null];
 };
 
-exports.packageType = constants.PACKAGE_TYPE_OS;
+exports.packageType = constants.PACKAGE_TYPE_I;
diff --git a/DocService/sources/DocsCoServer.js b/DocService/sources/DocsCoServer.js
index b654c9c..31bdf2d 100644
--- a/DocService/sources/DocsCoServer.js
+++ b/DocService/sources/DocsCoServer.js
@@ -165,7 +165,7 @@ let connections = []; // Активные соединения
 let lockDocumentsTimerId = {};//to drop connection that can't unlockDocument
 let pubsub;
 let queue;
-let licenseInfo = {type: constants.LICENSE_RESULT.Error, light: false, branding: false, customization: false, plugins: false};
+let licenseInfo = license.readLicense().next().value[0];
 let licenseOriginal = null;
 let shutdownFlag = false;
 let expDocumentsStep = gc.getCronStep(cfgExpDocumentsCron);
@@ -3328,7 +3328,7 @@ exports.install = function(server, callbackFunction) {
   });
 };
 exports.setLicenseInfo = function(data, original ) {
-  licenseInfo = data;
+  logger.debug('Not updating license info', data);
   licenseOriginal = original;
 };
 exports.getLicenseInfo = function() {
diff --git a/DocService/sources/server.js b/DocService/sources/server.js
index ccc232b..7c4e80d 100644
--- a/DocService/sources/server.js
+++ b/DocService/sources/server.js
@@ -147,7 +147,6 @@ try {
 } catch (e) {
 	logger.warn('Failed to subscribe to plugin folder updates. When changing the list of plugins, you must restart the server. https://nodejs.org/docs/latest/api/fs.html#fs_availability');
 }
-fs.watchFile(configCommon.get('license').get('license_file'), updateLicense);
 setInterval(updateLicense, 86400000);
 
 // Если захочется использовать 'development' и 'production',
diff --git a/FileConverter/sources/convertermaster.js b/FileConverter/sources/convertermaster.js
index 2c4526f..ec07464 100644
--- a/FileConverter/sources/convertermaster.js
+++ b/FileConverter/sources/convertermaster.js
@@ -85,7 +85,6 @@ if (cluster.isMaster) {
 
   updateLicense();
 
-  fs.watchFile(configCommon.get('license').get('license_file'), updateLicense);
   setInterval(updateLicense, 86400000);
 } else {
   const converter = require('./converter');
diff --git a/Makefile b/Makefile
index e8e1308..23f7e2e 100644
--- a/Makefile
+++ b/Makefile
@@ -87,7 +87,7 @@ DEBUG = $(BRANDING_DIR)/debug.js
 .PHONY: all clean install uninstall build-date
 
 .NOTPARALLEL:
-all: $(SPELLCHECKER_DICTIONARIES) $(TOOLS) $(SCHEMA) $(CORE_FONTS) $(DOCUMENT_TEMPLATES) $(LICENSE) $(WELCOME) $(INFO) build-date
+all: $(SCHEMA) $(LICENSE) $(WELCOME) $(INFO) build-date
 
 build-date: $(GRUNT_FILES)
 	sed "s|\(const buildVersion = \).*|\1'${PRODUCT_VERSION}';|" -i $(COMMON_DEFINES_JS)
