diff --git a/apps/documenteditor/mobile/src/lib/patch.jsx b/apps/documenteditor/mobile/src/lib/patch.jsx
index 963aca451..1ee48e856 100644
--- a/apps/documenteditor/mobile/src/lib/patch.jsx
+++ b/apps/documenteditor/mobile/src/lib/patch.jsx
@@ -4,7 +4,7 @@ const EditorUIController = () => {
 };
 
 EditorUIController.isSupportEditFeature = () => {
-    return false
+    return true
 };
 
 EditorUIController.getToolbarOptions = () => {
